pystan/model.py

```diff
@@ -88,6 +97,9 @@ def _map_parallel(function, args, n_jobs):
             pool.join()
     else:
         map_result = list(map(function, args))
+    # reset STAN_NUM_THREADS
```

Comment:
Did you forget to remove this?

Reply:
I left it there because we edit env var.

Not really happy about it either. Can we edit stanfit.hpp to enable some specific flag?